tools/compare.go

```diff
@@ -17,64 +17,137 @@ package tools
 import (
 	"encoding/json"
 	"fmt"
+	"github.com/google/go-cmp/cmp"
+	"github.com/streamingfast/cli"
+	"github.com/streamingfast/substreams/block"
+	"google.golang.org/protobuf/proto"
 	"io"
-	"strconv"
+	"math"
 	"sync"
 
-	"github.com/google/go-cmp/cmp"
 	"github.com/spf13/cobra"
 	"github.com/streamingfast/bstream"
 	"github.com/streamingfast/dstore"
 	pbeth "github.com/streamingfast/firehose-ethereum/types/pb/sf/ethereum/type/v2"
 )
 
 var compareBlocksCmd = &cobra.Command{
-	Use:   "compareblocks <blocks_store_a> <blocks_store_b> <prefix>",
-	Short: "Checks for any differences between merge files of two different stores. (To compare the output of two different instrumentations, for example)",
-	Args:  cobra.ExactArgs(3),
-	RunE:  compareBlocksE,
+	Use:   "compare-blocks <expected_bundle> <actual_bundle> [<block_range>]",
+	Short: "Checks for any differences between merge files of two different stores. (To compare the output of two different instrumentation's, for example)",
+	Long: cli.Dedent(`
+		compare-blocks takes in two stores of merged blocks and a range specifying the first x blocks you want to compare.
+		It operates in chunks of 100,000 blocks. Chunks that contain a difference will be communicated as well as the blocks within 
+		that contain differences. Chunks that do not have any differences will be outputted as identical.
+		
+		After passing through the chunks, it will output instructions on how to locate a specific difference based on the
+		blocks that were given. This is done by applying the --diff=true flag before your args. 
```

Comment:
Normally `--diff` is what is used instead of `--diff=true`, boolean flag that needs activation are always presented using plain `--<flag>`